Header header
float32 cell_size      #how big is each cell
float32 x_cen          #cneter of the map
float32 y_cen          #cneter of the map
float32 z_cen          #cneter of the map
int32 x_size	       #number of x cells
int32 y_size	       #number of y cells
int32 z_size	       #number of z cells

int32[] data
